tests/unit/environments/test_nemo_gym.py

```
@@ -199,3 +202,108 @@ def _standardize(l: list[dict]):
+            for task in orig_run_examples(self, examples, head_server_config):
+                row, result = task
+                # insert a NaN in the generation_log_probs
+                has_generation_log_probs = False
+                for i in range(len(result["response"]["output"])):
+                    if "generation_log_probs" in result["response"]["output"][i]:
+                        result["response"]["output"][i]["generation_log_probs"][-1] = float('nan')
+                        has_generation_log_probs = True
+                        break
+
+                if not has_generation_log_probs:
+                    raise ValueError("No generation_log_probs found in the result")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)